### DISCUSSION TITLE
JSON tags in types.go package - release when?

### DISCUSSION THREAD

**Discussion by tender-barbarian** [vmware/govmomi#3135] [Community Help] [answered]
Hi all!

I'm working on a project which collects events from vCenter and pushes them to our log collector. As such, we need a way to marshall events returned from API into JSON. Current release v0.30.4 does not include JSON tag for event types defined in `vim25/types.go`.  But I can see that it's been added in the main branch: https://github.com/vmware/govmomi/commit/99200868c514fe9560c52c2dcb6db484ca6a5e8e

So I guess question is, when is this going to make it's way into release? Currently we are using main branch of govmomi to have this change implemented, but it's not the best way to go, of course.

**Comment by paveljanda** [comment]
Hi, guys. Can you pls have a look at that? Without that, we have to realy on either an old release (0.28) or on main branch HEAD which is not really a suitable solution for production use.


Thanks!

**Comment by tender-barbarian** [comment]
Can anyone have a look pls? We really need this commit to be added to latest release.

**Reply by dougm** [reply, marked as answer]
Hi folks, json tags are included in v0.31.0, released today: https://github.com/vmware/govmomi/releases/tag/v0.31.0

**Reply by tender-barbarian** [reply]
Many thanks!

**Reply by paveljanda** [reply]
Thanks, cool!
